README.md

```diff
@@ -0,0 +1,317 @@
+# s3nd - The Deliverator
+
+![The Deliverator](./images/2b93b4eb-4971-4a1d-b64d-fc5ec046a06c.png)
+
+> The S3 Nexus Deliverator -- "s3nd" to the mortals who tap its command and
+> watch bytes ignite—is no ordinary file-pusher; it is a swaggering,
+> carbon-fiber katana forged for the cloud-age ronin who moves data at orbital
```

Comment:
"it" and "who" sound funny together.  Maybe "that" rather than "who" ?

Reply:
I think it works either way but I'll adopt the recommendation.